examples/plot_bch_so3.py

```diff
@@ -19,7 +19,7 @@
 
 import geomstats.backend as gs
 from geomstats.geometry.skew_symmetric_matrices import SkewSymmetricMatrices
-from geomstats.geometry.special_orthogonal import SpecialOrthogonal
+from geomstats.geometry.so3 import SpecialOrthogonal
```

Comment:
Naming inconsistency in the .py files:
- special_orthogonal.py
versus
- so3.py

We should:
- either have special_orthogonal and special_orthogonal_3,
- or have son and so3.

What do you prefer? If we move to `SO`, then the objects should probably also be called `SO` and `SO3`, and then we should probably rename GeneralLinear to GL, etc etc.

I would slightly prefer staying very explanatory at first (keep special_orthogonal.py, create special_orthogonal_3.py) - so that non-mathematician users have clear keywords to google if they don't know these names - but happy to hear others opinions. @opeltre @nguigs @pchauchat ?

Reply:
Non-mathematician people will most probably look for "3D rotations", which they would find by googling either name I think. The only thing is that, if at some point you decide to create files for the Lie algebras, `so3` will be confusing, but that's a little far-fetched.